### PR TITLE
[148] export student units as csv

### LINKED ISSUE #148: Export instructor's students view as csv
### Description

Currently, there's no easy way to export the student progress into a workable format such as a CSV. It would be very useful to be able to export the students' information and progress in a file. 

### Functional Spec

The information feeding the [Instructor's students view page](https://github.com/LDSSA/portal/blob/master/portal/academy/views.py#L123) needs to be parsed and provided in simple text format.

### Acceptance criteria

- [ ] A button triggering the export of the table information into a file

### PR DESCRIPTION
**Pull request by CatarinaSilva**
Closes #148 

ezgif-2-02ea555b8c0e

### CODE REVIEW

**Comment by cimendes** [comment]
Addressed #148

**Comment by cimendes** [comment]
I am in awe ❤️

File: portal/academy/views_test.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import pytest
```
Comment (hcastilho) [inline portal/academy/views_test.py:1]: Shouldn't this be placed in a `tests/` dir to follow the pattern of the rest of the project?

File: portal/academy/views.py [hunk 2]
```diff
@@ -200,14 +227,19 @@ def get(self, request, *args, **kwargs):
...
-            object_list=object_list,
-            spc_list=spc_list,
-            unit_list=unit_list,
-            max_score=max_score,
-            workspace_url=settings.SLACK_WORKSPACE,
-        )
-        return self.render_to_response(context)
+        if "download" in kwargs and kwargs["download"] == "csv":
+            response = HttpResponse(csvdata(spc_list, unit_list, object_list), content_type="text/csv")
+            response["Content-Disposition"] = "attachment; filename=test.csv"
```
Comment (hcastilho) [inline portal/academy/views.py:232]: I would suggest a more descriptive filename.

File: portal/academy/views.py [hunk 3]
```diff
@@ -48,6 +50,7 @@ def get_grade(unit, user):
     return grade
 
 
+
```
Comment (hcastilho) [inline portal/academy/views.py:53]: 3 blank lines

**Review by hcastilho — CHANGES_REQUESTED** [review]

File: portal/academy/views_test.py (hunk 1 above)
Comment (CatarinaSilva) [reply portal/academy/views_test.py:1]: Different habits 😅 I'll move it

Comment (CatarinaSilva) [reply portal/academy/views_test.py]: Done

File: portal/academy/views.py (hunk 2 above)
Comment (CatarinaSilva) [reply portal/academy/views.py]: Changed to `student-grades.csv`

File: portal/academy/views.py (hunk 3 above)
Comment (CatarinaSilva) [reply portal/academy/views.py]: Cleaned

**Review by hcastilho — APPROVED** [review]

**Pull request merged**